LiteCore/RevTrees/RawRevTree.cc

```diff
@@ -64,23 +65,38 @@ namespace litecore {
         while ( entry < (const void*)raw_tree.end() ) {
             RevTree::RemoteID remoteID = endian::dec16(entry->remoteDBID_BE);
             auto              revIndex = endian::dec16(entry->revIndex_BE);
+            if ( remoteID == 0 && revIndex == 0 ) {
+                ++entry;
+                break; // The zero mark
```

Comment:
I don't know about the structure of the rev-tree. What is the zero mark? Is this new?

Reply:
This definitely needs to be documented in comments.  Don't just add something artificial without an explanation.